### ISSUE TITLE
Delete Repo?

### ISSUE THREAD

**Issue by RichardLitt** [ipfs/js-ipfs-path#2]
Doesn't seem like it has been or will be used.

**Comment by diasdavid** [comment]
yeah, there is nothing here.

@mappum do you still want to use this for anything?

**Comment by mappum** [comment]
Nope, feel free to nuke it. Also, let me know if I should add you as collaborators on NPM so you can use the name for something else.

**Comment by RichardLitt** [comment]
No worries for now. If we need it we'll ping. :)
